_examples/eks/eks-cluster/cluster.tf

```diff
@@ -22,8 +22,8 @@ resource "aws_eks_node_group" "k8s-acc" {
   subnet_ids      = aws_subnet.k8s-acc.*.id
 
   scaling_config {
-    desired_size = 1
-    max_size     = 1
+    desired_size = 3
+    max_size     = 3
```

Comment:
I increased this because the acceptance tests couldn't run with so few nodes. It was unable to schedule my pods.